packages/node_modules/@webex/react-component-activity-text/src/index.test.js

```diff
@@ -20,6 +21,12 @@ describe('ActivityText component', () => {
     expect(component).toMatchSnapshot();
   });
 
+  it('renders clean links properly', () => {
+    const component = renderer.create(<ActivityText content={cleanLink} />);
+
+    expect(component).toMatchSnapshot();
+  });
```

Comment:
Can we also have tests for cases where there is already `_blank` or `mailto:` or `tel:` in the anchor tag?

Reply:
Added tests